lib/rototiller/utilities/env_var.rb

```diff
@@ -67,6 +73,11 @@ def message=(message)
     reset
   end
 
+  def required=(required)
+    @required = required
+    reset
+  end
```

Comment:
Added a comment above this block of variable assignment methods that are all doing the same thing.